packages/cosmic-proto/src/address-hooks.js

```diff
@@ -31,6 +31,10 @@
 import { bech32 } from 'bech32';
 import queryString from 'query-string';
 
+/** support environments without SES via Object.freeze */
+const hardenOrFreeze =
+  harden && typeof harden === 'function' ? harden : Object.freeze;
```

Comment:
since `harden` may not be defined,
```suggestion
const hardenOrFreeze = typeof harden === 'function' ? harden : Object.freeze;
```

Reply:
Thanks these changes are now in https://github.com/Agoric/agoric-sdk/pull/10683